backend/dristi-services/case/src/main/resources/db/migration/main/V20260427150000__lifecycle_status__ddl.sql

```
@@ -0,0 +1,16 @@
+ALTER TABLE dristi_cases ADD COLUMN lifecycleStatus varchar(32) DEFAULT 'ACTIVE';
+
+-- Migrate existing data from isLPRCase boolean to lifecycleStatus enum
+UPDATE dristi_cases SET lifecycleStatus = 'LPR' WHERE isLPRCase = true;
+UPDATE dristi_cases SET lifecycleStatus = 'ACTIVE' WHERE isLPRCase = false OR isLPRCase IS NULL;
+
+-- For LPR cases, move stageBackup to stage
+UPDATE dristi_cases SET stage = stagebackup WHERE isLPRCase = true AND stagebackup IS NOT NULL;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)